org.bluesabre.MenuLibre.yaml

```
@@ -0,0 +1,107 @@
+app-id: org.bluesabre.MenuLibre
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: menulibre-wrapper
+rename-icon: menulibre
+rename-desktop-file: menulibre.desktop
+finish-args:
+  - '--filesystem=host:ro'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=ipc'
+  - '--filesystem=xdg-data/applications:create'
+  - '--filesystem=~/.local/share/flatpak:ro'
```

```suggestion
  - '--filesystem=xdg-data/applications:create'
  - '--filesystem=xdg-data/flatpak:ro'
  - '--filesystem=/var/lib/flatpak:ro'
```

`--filesystem=xdg-data/flatpak:ro` is more standard compliant than `~/.local/share/flatpak`

`--filesystem=/var/lib/flatpak:ro` allows reading system flatpaks (host:ro doesn't cover it IIRC)